src/Meshes/interval.jl

```diff
@@ -305,7 +310,7 @@ function IntervalMesh(
     h =
         h_bottom .+
         (ζ_n .- ζ_n[1]) * (h_top - h_bottom) / (ζ_n[end - 1] - ζ_n[1])
-    faces = (z_bottom + (z_top - z_bottom)) * exp_stretch.(ζ_n, h)
+    faces = z_bottom .+ (z_top - z_bottom) * exp_stretch.(ζ_n, h)
```

Comment:
@charleskawczynski Here is a bug fix. Would this be a breaking change? This technically changes the behavior of the vertical mesh, but since we always use z_bottom = 0 in ClimaAtmos, it wouldn't change the results. Just want to make sure if the next climacore release should be a minor or patch release.